### PR TITLE
fixes #35136

### LINKED ISSUE #35136: Typo on doc
Line 194 is missing the square bracket:

- **wrong**
`Use distribution to create a linear combination of value with shape batch_size, Tq, dim]:`

- **correct**
`Use distribution to create a linear combination of value with shape [batch_size, Tq, dim]:`

Link to the line code:
https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/layers/dense_attention.py#L194

### PR DESCRIPTION
**Pull request by ymodak**
fixes #35136

### CODE REVIEW

**Review by SumanSudhir — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
We cannot accept PRs to `r2.0` or other release branches after the corresponding release has finished, except if they are related to a patch release for security.

**Comment by mihaimaruseac** [comment]
I tried changing base to `master` but there are some conflicting commits so it would need your work to rebase. Please reopen against `master`
